### PR TITLE
adding ansible file

### PR DESCRIPTION
**Pull request by vij-pixel**
Hi @harish123azure ,

Can you review terraform code and approve PR.

Regards,
abcd

### CODE REVIEW

File: terraform-ec2.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+resource "aws_instance" "web" {
+  ami           = data.aws_ami.ubuntu.id
+  instance_type = "t3.micro"
```
Comment (harish123azure) [inline terraform-ec2.txt]: change instance type to t2.large

File: terraform-ec2.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+  tags = {
+    Name = "HelloWorld"
```
Comment (harish123azure) [inline terraform-ec2.txt]: change name to jenkins server

**Review by harish123azure — CHANGES_REQUESTED** [review]
Can you fix below comments

**Review by vij-pixel — COMMENTED** [review]
Modified code change as per review comments. updated PR

**Review by harish123azure — APPROVED** [review]
looks good

**Pull request merged**